fronts-client/src/constants/feastPalettes.ts

```diff
@@ -0,0 +1,278 @@
+  },
+  Picnic: {
+    lightForegroundHex: '#427585',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#ACD8E5',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/21/Picnic.png',
+  },
+  Potatos: {
+    lightForegroundHex: '#99614B',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#F0C5B4',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/21/Potatos.png',
+  },
+  Rice: {
+    lightForegroundHex: '#68773C',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#DAE5B8',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/21/Rice.png',
+  },
+  Salads: {
+    lightForegroundHex: '#697311',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#E1E5B8',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/21/Salads.png',
+  },
+  'Quick Meals': {
+    lightForegroundHex: '#994100',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#ACD8E5',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/21/QuickmealsV2-01.png',
+  },
+  Soup: {
+    lightForegroundHex: '#406508',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#DAE5B8',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/21/Soups.png',
+  },
+  Spring: {
+    lightForegroundHex: '#697311',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#E1E5B8',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/21/Spring.png',
+  },
+  Summer: {
+    lightForegroundHex: '#9A1E1E',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#F0BAB4',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/21/Summer.png',
+  },
+  Sprouts: {
+    lightForegroundHex: '#9A1E1E',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#F0BAB4',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/01/Sprouts-01-01.png',
+  },
+  Tacos: {
+    lightForegroundHex: '#DB2712',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#FCF1F0',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/21/Tacos.png',
+  },
+  Tins: {
+    lightForegroundHex: '#427585',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#ACD8E5',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/21/Tins.png',
+  },
+  Traybake: {
+    lightForegroundHex: '#801919',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#FCF1F0',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/21/Traybake.png',
+  },
+  'TV snacks': {
+    lightForegroundHex: '#427585',
+    lightBackgroundHex: '#F9F9F5',
+    darkForegroundHex: '#ACD8E5',
+    darkBackgroundHex: '#363632',
+    imageURL: 'https://uploads.guim.co.uk/2024/03/21/TVsbacks.png',
```

Comment:
Nitpick #2: TVs**n**acks?

Reply:
as above, but we could always rename 'em, it's somewhat irritating